liberty-maven-plugin/src/it/tests/appsdirectory-set-appz-notconfigured-it/pom.xml

```diff
@@ -0,0 +1,194 @@
+            </execution>
+            <execution>
+               <id>start-liberty-server</id>
+               <phase>pre-integration-test</phase>
+               <goals>
+                   <goal>start-server</goal>
+               </goals>
+               <configuration>
+                   <background>true</background>
+                   <verifyTimeout>40</verifyTimeout>
+               </configuration>
+            </execution>
+            <execution>
+               <id>check-liberty-server</id>
+               <phase>pre-integration-test</phase>
+               <goals>
+                   <goal>server-status</goal>
+               </goals>
+            </execution>
+            <execution>
+                <id>stop-liberty-server</id>
+                <phase>post-integration-test</phase>
+                <goals>
+                    <goal>stop-server</goal>
+                </goals>
+            </execution>
+            <execution>
+                <id>clean-server</id>
+                <phase>post-integration-test</phase>
+                <goals>
+                    <goal>clean-server</goal>
+                </goals>
+                <configuration>
+                    <cleanDropins>true</cleanDropins>
+                    <cleanApps>true</cleanApps>
+                </configuration>
+            </execution>
+         </executions>
+       </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <configuration>
+                    <redirectTestOutputToFile>true</redirectTestOutputToFile>
+                    <forkMode>once</forkMode>
+                    <forkedProcessTimeoutInSeconds>300</forkedProcessTimeoutInSeconds>
+                    <argLine>-enableassertions</argLine>
+                    <workingDirectory>${project.build.directory}</workingDirectory>
+                    <includes>
+                        <include>**/*Test.java</include>
+                    </includes>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>integration-test</id>
+                        <goals>
+                            <goal>integration-test</goal>
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>verify</id>
+                        <phase>install</phase>
+                        <goals>
+                            <goal>verify</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <skip>true</skip>
+                </configuration>
+            </plugin>
+    </plugins>
+  </build>
+</project>
```

Comment:
keep this test, but also let's go through the exercise of creating a liberty-assembly project to have a much more simplified pom.  Also, you could use test-start-server and test-stop-server here or remove them all together since I don't think your test needs a server started does it?